dashboard/app/models/section.rb

```diff
@@ -60,12 +60,12 @@ def workshop_section?
     Pd::Workshop::SECTION_TYPES.include? section_type
   end
 
-  validates_presence_of :user
+  validates_presence_of :user, unless: -> {deleted?}
   def user_must_be_teacher
     return unless user
     errors.add(:user_id, 'must be a teacher') unless user.teacher?
   end
-  validate :user_must_be_teacher
+  validate :user_must_be_teacher, unless: -> {deleted?}
```

Comment:
Though this is a validation we'd like to keep... If the follower is deleted, the teacher may be deleted, in which case `user` is `nil` (scoped to non-deleted accounts), meaning `user.teacher?` would throw.